apps/dashboard/src/routes/org/[slug]/pathways/+page.svelte

```diff
@@ -0,0 +1,103 @@
+<script lang="ts">
+  import { Search, Dropdown } from 'carbon-components-svelte';
+  import PrimaryButton from '$lib/components/PrimaryButton/index.svelte';
+  import { courseMetaDeta } from '$lib/components/Courses/store';
+  import { Add } from 'carbon-icons-svelte';
+  import { isMobile } from '$lib/utils/store/useMobile';
+  import { currentOrgPath, isOrgAdmin } from '$lib/utils/store/org';
+  import { t } from '$lib/utils/functions/translations';
+  import { VARIANTS } from '$lib/components/PrimaryButton/constants';
+  import IconButton from '$lib/components/IconButton/index.svelte';
+  import Grid from 'carbon-icons-svelte/lib/Grid.svelte';
+  import List from 'carbon-icons-svelte/lib/List.svelte';
+  import { onMount } from 'svelte';
+  import { goto } from '$app/navigation';
+  import NewPathwayModal from '$lib/components/Org/PathWay/NewPathwayModal.svelte';
+  import Pathway from '$lib/components/Org/PathWay/Pathway.svelte';
+
+  export let data;
+
+  let { cantFetch } = data;
+  let searchValue = '';
+  let selectedId: string = '0';
+  let collections = [1, 2, 3, 4, 5];
+
+  const setViewPreference = (preference: 'grid' | 'list') => {
+    $courseMetaDeta.view = preference;
```

Comment:
Why is the name of this store `courseMetaDeta` when this is a pathway feature?

Reply:
This was a placeholder for the pathways since i wasn't sure of the data structure that the pathway data was to take but the logic was implemented in PR #408

Reply:
No need, that's fine